OMX-27.ino

```diff
@@ -1772,7 +1889,7 @@ void playNote(int patternNum) {
 	case STEPTYPE_PLAY:	// regular note on
 		seq_velocity = stepNoteP[playingPattern][seqPos[patternNum]].vel;
 		
-		pendingNoteOffs.insert(stepNoteP[patternNum][seqPos[patternNum]].note, PatternChannel(patternNum), micros()+ ( stepNoteP[patternNum][seqPos[patternNum]].len + 1 )*step_micros);
+		pendingNoteOffs.insert(stepNoteP[patternNum][seqPos[patternNum]].note, PatternChannel(patternNum), micros()+ ( (stepNoteP[patternNum][seqPos[patternNum]].len + 1 )*step_micros)*.80); // 90% to account for jitter and avoid overlaps
```

Comment:
@okyeron this i believe will change back w/that bug you discovered.. i'll wait for that to be merged to test reverting this

Reply:
I think part of your midi overlap was that notelen was improperly initialized to 1 so `.len` above is actually 2 steps instead of 1

See the most recent commit on main for fix